events.go

```diff
@@ -49,9 +49,21 @@ type PushEvent struct {
 		WebURL            string          `json:"web_url"`
 		Visibility        VisibilityValue `json:"visibility"`
 	} `json:"project"`
-	Repository        *Repository `json:"repository"`
-	Commits           []*Commit   `json:"commits"`
-	TotalCommitsCount int         `json:"total_commits_count"`
+	Repository *Repository `json:"repository"`
+	Commits []*struct {
+		ID        string     `json:"id"`
+		Message   string     `json:"message"`
+		Timestamp *time.Time `json:"timestamp"`
+		URL       string     `json:"url"`
+		Added     []string   `json:"added"`
+		Modified  []string   `json:"modified"`
+		Removed   []string   `json:"removed"`
+		Author struct {
+			Name  string `json:"name"`
+			Email string `json:"email"`
+		} `json:"author"`
```

Comment:
Would be great if you can keep the same order as in the docs. So that would place `Author` between `URL` and `Added`.